sdk/build.gradle

```diff
@@ -80,6 +81,13 @@ uploadArchives {
 dependencies {
     compile 'joda-time:joda-time:2.9.2'
     compile 'com.squareup.okhttp3:okhttp:3.4.1'
+    compile 'com.android.support:appcompat-v7:26.1.0'
```

Comment:
Can we do this without support library? We try to avoid that for the compatibility issue with our client.

Reply:
This one using for testing with `AppCompatActivity`. Btw I can change to normal `Activity` and delete support library.

Reply:
@pitiphong-p please review gain krub.